### PR TITLE
Add Rawls submission option for delete_intermediate_output_files WA-98

### PR DESCRIPTION
**Pull request by kshakir**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

**Comment by kshakir** [comment]
Manually run swatomation pipeline:
✅https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/30218/

File: serviceTest/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 
 This file documents changes to the `workbench-service-test` library, including notes on how to upgrade to new versions.
 
+## 0.17
+
+SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-service-test" % "0.17-TRAVIS-REPLACE-ME"`
```
Comment (aednichols) [inline serviceTest/CHANGELOG.md:7]: Huh? Does Travis replace strings in the CHANGELOG?

**Review by aednichols — APPROVED** [review]

File: serviceTest/CHANGELOG.md (hunk 1 above)
Comment (kshakir) [reply serviceTest/CHANGELOG.md:7]: Yep. It's even referred to in the pull-request-template message.

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala [hunk 2]
```diff
@@ -60,6 +61,7 @@ trait Rawls extends RestClient with LazyLogging {
 
   }
 
+  //noinspection RedundantBlock,ScalaUnnecessaryParentheses
```
Comment (salonishah11) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala:64]: What does this do?

Comment (kshakir) [reply serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Rawls.scala:64]: It shuts off warnings about these issues in the file. It then allows other bugs to be quickly discovered.

- These warnings are enabled by default in IntelliJ IDEA
- This switches them off only for this file
- Doesn't muddle this PR with other fixes for now
- Suppression can be removed easily and issues cleaned up in a later PR
- Unmasks more important inspections that catch bugs like `if (myString == myOption)` which will never match
- https://www.jetbrains.com/help/idea/code-inspection.html

In this case, this class contains extra `{` and extra `(` that don't need to be there.

**Review by salonishah11 — APPROVED** [review]
I prefer `delete_intermediate_output_files` over `delete_intermediates` because `delete_intermediate_output_files` suggests that we are **only** deleting the intermediate output files (which is the current implementation) and leaving files like stderr, stdout, rc, transfer scripts, etc. created by Cromwell as it is in the bucket. While `delete_intermediates` suggests that we are deleting everything except final outputs. That being said, I am not married to it being called `delete_intermediate_output_files` and am open to it being renamed :)

**Pull request merged**